math/quadratic_equations_complex_numbers.cpp

```diff
@@ -0,0 +1,166 @@
+/**
+ * @file
+ * @brief Calculate quadratic equation with complex roots, i.e. b^2 - 4ac < 0.
+ * @see https://en.wikipedia.org/wiki/Quadratic_equation,
+ * https://en.wikipedia.org/wiki/Quadratic_equation#discriminant
+ *
+ * @author [Renjian-buchai](https://github.com/Renjian-buchai)
+ *
```

Comment:
```suggestion
 * @description
```
Seems like this is a description